src/components/common/FilterGroup/index.tsx

```diff
@@ -60,9 +60,13 @@ export default function FilterGroup(props: FilterGroupProps): JSX.Element {
   const [filters, setFilters] = useState<Record<string, SearchNodePayload>>(initialFilters);
 
   const onFilterChange = (key: string, filter: SearchNodePayload) => {
-    const newFilt: Record<string, SearchNodePayload> = {};
-    newFilt[key] = filter;
-    setFilters({ ...filters, ...newFilt });
+    if (filter.values.length) {
+      const newFilt: Record<string, SearchNodePayload> = {};
```

Comment:
i understand it's old code but since we are touching this.. might as well call it newFilter?

Reply:
I changed it to:
```typescript
setFilters({ ...filters, [key]: filter });
```